test/utils/image/manifest.go

```diff
@@ -68,15 +68,15 @@ func (i *Config) SetVersion(version string) {
 	i.version = version
 }
 
-func Init(repoList string) {
-	registry, imageConfigs, originalImageConfigs = readRepoList(repoList)
+func Init(repoList, imageRepository string) {
+	registry, imageConfigs, originalImageConfigs = readRepoList(repoList, imageRepository)
```

Comment:
and this overrides the global variable that is initialized with the environment variable??